state-chain/chains/src/btc/api.rs

```diff
@@ -89,7 +89,7 @@ where
 				.ok_or(AllBatchError::NotRequired)?,
 			number_of_outputs: (btc_outputs.len() + 1) as u64, // +1 for the change output
 		})
-		.ok_or(AllBatchError::Other)?;
+		.ok_or(AllBatchError::InvalidUtxo)?;
```

Comment:
I don't think this is an accurate name for this error.

I would call it UtxoSelectionFailed or UnableToSelectUtxos.